src/modules/Library/features/Applet/Item/Item.styles.ts

```diff
@@ -0,0 +1,95 @@
+import MdEditor from 'md-editor-rt';
+import { styled } from '@mui/system';
+import { Box } from '@mui/material';
+
+import { theme, variables, StyledFlexColumn, StyledFlexTopStart } from 'shared/styles';
+
+export const StyledItemContainer = styled(StyledFlexColumn)`
+  display: grid;
+  grid-template-columns: 4.8rem auto;
+  align-items: start;
+  padding: ${theme.spacing(0.8, 2.4)};
+
+  &:not(:last-child) {
+    border-bottom: ${variables.borderWidth.md} solid ${variables.palette.outline_variant};
+  }
+
+  .MuiFormControlLabel-root {
+    margin: 0;
+  }
+`;
+
+export const StyledItemHeader = styled(Box)`
+  display: grid;
+  grid-template-columns: 4.8rem auto;
```

Comment:
this style is the same as for the row 9, can be moved into a variable (the same styles in file `Activity.styles.ts`)